src/app/core/shared/resource-type.ts

```diff
@@ -1,28 +1,4 @@
-export enum ResourceType {
-  DSpaceObject = 'dspaceobject',
-  Bundle = 'bundle',
-  Bitstream = 'bitstream',
-  BitstreamFormat = 'bitstreamformat',
-  Item = 'item',
-  Collection = 'collection',
-  Community = 'community',
-  EPerson = 'eperson',
-  Group = 'group',
-  ResourcePolicy = 'resourcePolicy',
-  MetadataSchema = 'metadataschema',
-  MetadataField = 'metadatafield',
-  Relationship = 'relationship',
-  RelationshipType = 'relationshiptype',
-  ItemType = 'entitytype',
-  License = 'license',
-  Workflowitem = 'workflowitem',
-  Workspaceitem = 'workspaceitem',
-  SubmissionDefinitions = 'submissiondefinitions',
-  SubmissionDefinition = 'submissiondefinition',
-  SubmissionForm = 'submissionform',
-  SubmissionForms = 'submissionforms',
-  SubmissionSections = 'submissionsections',
-  SubmissionSection = 'submissionsection',
-  ClaimedTask = 'claimedtask',
-  PoolTask = 'pooltask'
+export class ResourceType {
```

Comment:
Can we add TypeDocs here too?  This is another key change in this PR, so we should document what this class represents (as it used to be an enum).